### PR TITLE
Change herd.herd to VARCHAR(10) from INTEGER

### PR DESCRIPTION
**Pull request by andkaha**
This allows us to store the herd IDs with a "G" or "M" prefix in the
database directly.

### CODE REVIEW

File: app/requirements.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ MarkupSafe==1.1.1
 mccabe==0.6.1
 peewee==3.13.2
 psycopg2-binary==2.8.5
-git+git://github.com/jameshicks/pydigree@master
+#git+git://github.com/jameshicks/pydigree@master
```
Comment (jhagberg) [inline app/requirements.txt]: Why do you comment this line is not related to the database changes?

**Review by jhagberg — CHANGES_REQUESTED** [review]

File: app/requirements.txt (hunk 1 above)
Comment (andkaha) [reply app/requirements.txt]: This was not supposed to be committed. It was a local change that I needed to be able to load the data (the `pydigree` Python module did not install on my system).  I will revert this change.

**Review by norling — APPROVED** [review]
Looks reasonable!

**Review by jhagberg — APPROVED** [review]
LGTM
